Sources/MapboxNavigation/RoutesPresentationStyle.swift

```diff
@@ -8,14 +10,17 @@ public enum RoutesPresentationStyle {
     
     /**
      Only first route will be presented on a map view.
+     
+     - parameter cameraOptions: The custom `CameraOptions` that the map view ill be presented with.
```

Comment:
```suggestion
     - parameter cameraOptions: The custom `CameraOptions` that the map view will use while presenting a single route.
```

Reply:
Thanks for the suggestion, the documentation is improved.